halfshell/source_s3.go

```diff
@@ -69,7 +69,9 @@ func (s *S3ImageSource) GetImage(request *ImageSourceOptions) (*Image, error) {
 }
 
 func (s *S3ImageSource) signedHTTPRequestForRequest(request *ImageSourceOptions) *http.Request {
-	imageURLPathComponents := strings.Split(request.Path, "/")
+	path := s.Config.PathPrefix + request.Path
```

Comment:
I will find time this week to do it. Or if you want to update it yourself and merge, I’m good with this :wink: 

The resulting source code will be much simpler with the directory setting.